controls/bsi_app_4_4.yml

```diff
@@ -404,10 +404,15 @@ controls:
     levels:
     - elevated
     description: >-
-      Pods SHOULD be stopped and restarted regularly if there is an increased risk of external
-      interference and a very high need for protection. No pod SHOULD run for more than 24
+      (1) Pods SHOULD be stopped and restarted regularly if there is an increased risk of external
+      interference and a very high need for protection.
+      (2) No pod SHOULD run for more than 24

```

Comment:
The default `podLifetime` is 24h, but it can be changed.
There could be a  rule to check that `podLifetime` is 24h.

Reply:
I have currently no idea how to check the podLifetime. By default there is no field fpr podLifetime in the yaml, also from what I understand the value can be formatted differently. So I have to either check for the field not existing at all, for a value less than "24h" (I think the type is time.Duration) or something like "86400s"

I would rather leave this as is for now and maybe expand on this later.